docs/tutorials/Aragon_Core_User_Guide.md

```diff
@@ -0,0 +1,197 @@
+## Aragon Core User Guide
+#### Documentation Version 1.0.5.1
+#### Aragon Core Version 0.5
+
+### 0. Prerequisites
+
+In addition to a working laptop or desktop computer with a decent internet connection, you will need the following to start using Aragon Core:
+
+- Up-to-date Brave, Chrome/Chromium, or Firefox web browser installed.
+- Up-to-date MetaMask browser extension installed. Use the "Old UI", not the “Beta UI”. If you are currently using the Beta UI, you can switch to the Old UI by toggling the UI switch on the Settings page of your MetaMask extension.
+- Test ether for the Ethereum Rinkeby test network. This test ether is used to pay for gas when creating and participating in DAOs. You can try the [Rinkeby faucet](https://faucet.rinkeby.io/) or post your Ethereum Rinkeby testnet address in the [#faucet channel on Aragon Chat](https://aragon.chat/faucet) to receive test ether for free. If you post your faucet request in the Aragon Chat, please wait patiently for an Aragon team member to see your message.
+
+### 1. Create a new Democracy DAO
+
+**Step 1. Navigate to [app.aragon.one](http://app.aragon.one) in your web browser and click “Create a new organization”**
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/welcome.png)
+
+**Step 2. Select “Token project with Democracy” then click “Next”**
+
+A “Token project with Democracy” is a DAO where the vote cast by a token holder is weighted proportionally to their ownership stake of DAO tokens. For example, if a tokenholder owns 5% of the tokens minted by the DAO then their vote will count as 5% of the total votes on a given proposal.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/choosedemo.png)
+
+**Step 3. Claim a domain name then click “Next”**
+
+Aragon Core uses the Ethereum Name System (ENS) to assign names to DAOs. This name currently cannot be changed, so choose wisely!
+
+TODO: note character limits
```

Comment:
Is this necessary to be here?

Reply:
I'm leaving it there until the TODO can be documented elsewhere (as an issue, for example)